Automattic/jetpack
# Connection JS package: new action to refresh the list of connected Plugins - apply it to My Jetpack

Fixes #23202 

#### Changes proposed in this Pull Request:

* Update the list of connected plugins after activating a product

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Activate and connect Jetpack
* Go to My Jetpack
* Click disconnect and look at the modal (do not disconnect)
* Confirm that only Jetpack is in the plugins list
* Cancel
* Activate Boost - Boost should be activated without a page reload
* Click on disconnect again
* Boost should be included in the list of active plugins
